components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -114,6 +114,7 @@ - (instancetype)initWithTitle:(NSString *)title message:(NSString *)message {
     _header.title = [title copy];
     _header.message = [message copy];
     self.backgroundColor = [UIColor whiteColor];
+    self.actionTintColor = [UIColor blackColor];
```

Comment:
Please don't use property setters in initializers. Directly access the autosynthesized ivar instead.  Please also use dot syntax for properties (on UIColor in this case).

```objc
_actionTintColor = UIColor.blackColor;
```